tsconfig.vitest.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "composite": true,
     "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.vitest.tsbuildinfo",
-    "lib": [],
+    "lib": ["DOM"],
```

Comment:
HTMLElement was not defined in testing files and was showing a lot of annoying errors